packages/office-ui-fabric-react/src/common/DocPage.types.ts

```diff
@@ -99,6 +99,9 @@ export interface IDocPageProps {
   /** Best practice as markdown string */
   bestPractices?: string;
 
+  /**Feedback tool */
```

Comment:
nit: Space before Feedback

It may also be good to expand on this comment a little. What's the feedback for?

Reply:
Strongly agree with Peter on expanding the comment.